presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -160,6 +160,7 @@
     private String temporaryTableSchema = "default";
     private HiveStorageFormat temporaryTableStorageFormat = ORC;
     private HiveCompressionCodec temporaryTableCompressionCodec = HiveCompressionCodec.SNAPPY;
+    private boolean createEmptyBucketFilesForTemporaryTable = true;
```

Comment:
It is fine to keep it "true" for the transitional period, but let's not forget to set it to "false" once the transition is done.